vuepress/docs/next/docs/getting-started/ent-svc.md

```diff
@@ -4,7 +4,9 @@ sidebarDepth: 1
 
 # Auxiliary Services
 
-The following sections discuss auxiliary services and commands.
+Auxiliary services extend the capabilities of a local development environment. Kubernetes can be configured to point to externally managed services, such as MySQL or PostgreSQL, or services are provided by the platform (e.g. Keycloak or RedHat SSO).
```

Comment:
Getting closer. Maybe this is even clearer "Auxiliary services add capabilities to a local environment to help accelerate application development. In a full Kubernetes setting, by comparison, the equivalent services are either provisioned by the Entando platform or the application can be configured to point to externally managed services."